packages/docusaurus/src/commands/swizzle.ts

```diff
@@ -18,24 +18,33 @@ export default async function swizzle(
   siteDir: string,
   themeName: string,
   componentName?: string,
+  typescript?: boolean,
 ): Promise<void> {
   const plugin = importFresh(themeName) as (
     context: LoadContext,
   ) => Plugin<unknown>;
   const context = loadContext(siteDir);
   const pluginInstance = plugin(context);
-  let fromPath = pluginInstance.getThemePath?.();
+  let fromPath = typescript
+    ? (pluginInstance.getTypeScriptThemePath ?? pluginInstance.getThemePath)?.()
+    : pluginInstance.getThemePath?.();
```

Comment:
I think we should fail fast here

I tried the following and expected it to fail: `yarn workspace docusaurus-2-website swizzle @docusaurus/theme-live-codeblock Playground --typescript`

but instead it did swizzle the components in JS.

Reply:
Fixed to avoid fallback to getThemePath